runtime/compiler/optimizer/J9EstimateCodeSize.cpp

```diff
@@ -1817,7 +1817,11 @@ TR_J9EstimateCodeSize::realEstimateCodeSize(TR_CallTarget *calltarget, TR_CallSt
                int32_t origRealSize = _realSize;
                int32_t origBigCalleesSize = _bigCalleesSize;
                bool prevNonColdCalls = _hasNonColdCalls;
-               bool estimateSuccess = estimateCodeSize(targetCallee, &callStack, /* recurseDown */ true, analyzedSizeThreshold);
+               bool estimateSuccess = false;
+               // Terminate recursive ECS along archetypes due to functional issues resulting from continuing ECS in Java 8 and 11.
```

Comment:
On the face of it, this seems like we won't inline past an archetype call, which could risk regressions on MH heavy workloads. I know this change has worked well in a non-MH heavy workload (Liberty) but have you tested this on MH heavy benchmarks ? If it has not regressed there, is there an intuitive reason for why ?